preditor/gui/suggest_path_quotes_dialog.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+
+from Qt.QtWidgets import QDialog
+
+from . import loadUi
+
+import logging
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
I'm guessing this was leftover from testing but, logging levels should not be set by random bits of code. It should only be set for a specific application. You should be able to use the logging toolbar icon in preditor to set this and the prefs should remember its setting. Please remove this line, though looks like you could remove the logger entirely as its no longer being used.

Reply:
Removed